_data/collection-index.yml

```
@@ -473,3 +473,8 @@
   contact: https://github.com/tomharvey/devcontainer-features/issues
   repository: https://github.com/tomharvey/devcontainer-features
   ociReference: ghcr.io/tomharvey/devcontainer-features
+- name: Azure Developer CLI Dev Container Feature
+  maintainer: Azure Developer CLI Team
+  contact: https://github.com/Azure/azure-dev/issues
+  repository: https://github.com/Azure/azure-dev
+  ociReference: ghcr.io/Azure/azure-dev
```

Hi 👋

The ORAS spec requires the `ociReference` to be in lower case. Can we update here, thanks?
The supporting tools do help convert the case, however, its better to sync as much as possible. 

```suggestion
  ociReference: ghcr.io/azure/azure-dev
```